### DISCUSSION TITLE
Create new note in current folder

### DISCUSSION THREAD

**Discussion by Arbiter313** [notaapp/talk#297] [Feature Requests]
It would be helpful to have a command to create a new note in the current folder, as opposed to just being able to create one in the inbox or root as is possible at present.

**Comment by erusev** [comment]
Do you mean setting `current folder` as your `Inbox folder`? Thanks!

<img width="358" alt="image" src="https://user-images.githubusercontent.com/184170/185911173-bd8c4c39-7142-4227-ba6f-1ebc9508afea.png">

**Reply by Arbiter313** [reply]
Ah no sorry. I mean having a similar command as in the screenshot below, but having the option to create the note in the folder you are currently working in instead of root/inbox. 

<img width="506" alt="Screenshot 2022-08-22 at 12 47 21 pm" src="https://user-images.githubusercontent.com/45447185/185914366-8e4a1591-f743-4705-8e05-447d235b826a.png">

**Reply by erusev** [reply]
Do you mean having a second create command in the ⌘N dialog? Otherwise, if you want this same command to save in a different folder you can use the `Inbox folder` option in preferences to control that, but at this point it doesn't support setting `current folder`.

**Reply by erusev** [reply]
Btw, in the ⌘N dialog, you can also press `Tab` to select a different folder, which defaults to the current folder, but note that at this point, this is an experimental behavior, and might change in future.

**Reply by Arbiter313** [reply]
Originally I did mean a second create command, but I have just tried the `Tab` behaviour and it is ideal for my use case so thank you! The only thing that could be improved with the `Tab` behaviour is being able to give the created note a custom title from the dialog itself instead of just creating an untitled note, but it definitely does the job.

**Reply by erusev** [reply]
You can type the name of the new note in the first step (where it says `Untitled`) -- doesn't this work for you? Thanks!

**Reply by Arbiter313** [reply]
Yes it does thank you!

**Reply by bellebethcooper** [reply]
I also didn't find the `Tab` option until reading this discussion, and I also didn't realise that I needed to name the new file before using `Tab` to select the folder, otherwise I would end up with an untitle file. I can now see (very clearly) where the title (or "Untitled") is shown after pressing `Tab` but somehow it was very easy to miss at first, so perhaps some extra work could be done to make this whole process more obvious? Or maybe just some more documentation (I've read the help docs a lot, sorry if I've missed it and it's already there).

**Reply by erusev** [reply]
Thanks for the feedback! We are experimenting with a few ideas to improve the create workflow.
